src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -312,20 +316,22 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
 
             docker {
                 name '${id}'
-                tags 'latest', 'another'
+                tags 'mytag', 'another'
```

Comment:
I think the latest tag is generated by default in docker < 1.10 - hence you get a conflict, so other than put logic in the test, I'm stumped as to what the solution would be to make this work pre 1.10